### PR TITLE
Source Salesforce: exclude KnowledgeArticleVersion from bulk api

### PR DESCRIPTION
**Pull request by edgao**
Followup to https://github.com/airbytehq/airbyte/pull/15901 - do the same thing for another stream

### CODE REVIEW

File: airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py [hunk 1]
```diff
@@ -138,6 +138,7 @@
     "EventWhoRelation",
     "FieldSecurityClassification",
     "KnowledgeArticle",
+    "KnowledgeArticleVersion",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py:141]: https://salesforce.stackexchange.com/a/71335 this answer suggests we should add the following entities too: 

```suggestion
    "KnowledgeArticleVersion",
    "KnowledgeArticleVersionHistory",
    "KnowledgeArticleViewStat",
    "KnowledgeArticleVoteStat",
```

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py (hunk 1 above)
Comment (edgao) [reply airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py:141]: ooh, good call. 

It looks like we're still missing these:
* List item (not sure if this is the name of an endpoint or some generic type of API object)
* LeadStatus
* OpportunityStage
* UserRecordAccess

For the last three, we don't yet support those streams - do you want to just add them in preemptively, or leave them until the streams are actually implemented?

Comment (sherifnada) [reply airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py:141]: happy to add them pre-emptively!

**Pull request merged**